### PR TITLE
Add Marr's Blessing quest

### PR DESCRIPTION
**Pull request by Vavehl**

### CODE REVIEW

File: rathemtn/Yilbit_Gerbigo.lua [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+end
+
+function event_trade(e)
+	local item_lib = require("items");
+	local fac = e.other:GetFaction(e.self);
+
+	if(item_lib.check_turn_in(e.trade, {item1 = 55035})) then -- Sir Bronthas's Remains
+		if(e.other:Race() ==  "Froglok" or fac < 5) then
+			e.self:Say("You actually found his remains?! You are remarkable. We must get Sir Bronthas a proper burial. As promised, here is your reward.");
+			e.other:SummonItem(55036); -- Marr's Blessing
```
Comment (fryguy503) [inline rathemtn/Yilbit_Gerbigo.lua:35]: Lets convert SummonItem, Ding and AddExp into the modern QuestReward

File: rathemtn/Yilbit_Gerbigo.lua [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		end
+	end
+end
+
+function event_trade(e)
+	local item_lib = require("items");
+	local fac = e.other:GetFaction(e.self);
+
+	if(item_lib.check_turn_in(e.trade, {item1 = 55035})) then -- Sir Bronthas's Remains
+		if(e.other:Race() ==  "Froglok" or fac < 5) then
```
Comment (fryguy503) [inline rathemtn/Yilbit_Gerbigo.lua:33]: Do the Race and faction check first so you do not need the else summonitem logic

File: rathemtn/Yilbit_Gerbigo.lua [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+		elseif(e.message:findi("assistance")) then
+			e.self:Say("They wish for us to venture to the castle and find any sign of what happened to Sir Bronthas. I am afraid however that I was wounded in the war with the Trolls and can no longer fight and Shin Master Grubbus has his hands full training the influx of new recruits. Perhaps you could find some time to [" .. eq.say_link("look into this") .. "] for us?");
+		elseif(e.message:findi("look into this")) then
+			e.self:Say("You are a true follower of Marr. However I worry for your safety. If you do accept this task, please make sure you are ready. The castle is a deadly place. Should you find any trace of Sir Bronthas's remains, please bring them back to me. There is apparently a reward for completing this. Marr's blessing be with you.");
+		end
+	else
+		if(e.message:findi("hail")) then
```
Comment (fryguy503) [inline rathemtn/Yilbit_Gerbigo.lua:22]: no need for the if hail check here, if not a Froglok or Faction any chat can just return the "Not something right" text

File: rathemtn/Yilbit_Gerbigo.lua [hunk 4]
```diff
@@ -0,0 +1,50 @@
+-- Yilbit Gerbigo
+-- Marr's Blessing
+-- items: 55035, 55036
+function event_say(e)
+	local fac = e.other:GetFaction(e.self);
+	
+	if(e.other:Race() ==  "Froglok" or fac < 5) then
+		if(e.message:findi("hail")) then
+			e.self:Say("Salutations. Come sit with me and have a drink. I am just going over some [" .. eq.say_link("business") .. "] right now.");
```
Comment (fryguy503) [inline rathemtn/Yilbit_Gerbigo.lua:9]: No need for any eq.say_link calls in Say types, The source code will autocovert [word] to saylinks (just not in Emotes)

Comment (Vavehl) [reply rathemtn/Yilbit_Gerbigo.lua:9]: Thanks for the tips -- I'll update the lua file with these suggestions and submit another pull request. :)

Comment (fryguy503) [reply rathemtn/Yilbit_Gerbigo.lua:9]: Sounds good, just commiting to the same local branch will add it to the current PR

Comment (Vavehl) [reply rathemtn/Yilbit_Gerbigo.lua]: Heh, I missed one of the say_link calls -- sorry about that. Going to clean it up and commit again.

Comment (fryguy503) [reply rathemtn/Yilbit_Gerbigo.lua]: lgtm - Thank you for the PR

**Pull request merged**